### PR TITLE
About my-self and works card feature

### PR DESCRIPTION
**Pull request by zewdu444**
Mobile first final Portfolio Page

- works showcase and about me feature has been  added 
- In addition to flex box html element organized by CSS grid.
- Works card for each showcase were styled 
- Additional button and a link element had hover and active properties
- IN About my-self images and  button style were implemented  
-Read me File broken link has been fixed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,40 +11,171 @@
...
+            </li>
+            <li>
+              <a href="#"><i class="fa-solid fa-hand-peace fa-lg"></i></a>
+            </li>
+          </ul>
+        </section>
+      </div>
+      <!-- project showcase -->
+      <!-- Tonic card one -->
+      <div class="workscard">
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use a section tag for the works container.

File: index.html [hunk 2]
```diff
@@ -11,40 +11,171 @@
...
+        <img src="./images/Tonic one.png" alt="Tonic" />
+        <p class="projecttitle">Tonic</p>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use a header tag hx for all the project titles.

File: index.html [hunk 3]
```diff
@@ -11,40 +11,171 @@
...
+          <li><button>html</button></li>
+          <li><button>css</button></li>
+          <li><button>javaScript</button></li>
+        </ul>
+        <button class="projectbutton">See Project</button>
+      </div>
+
+      <!-- about me -->
+
+      <div class="aboutmyself">
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use a section tag for the abiout me container.

File: index.html [hunk 4]
```diff
@@ -11,40 +11,171 @@
...
+          <p class="abouttitle">About Myself</p>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use a header tag hx for the title.

File: index.html [hunk 5]
```diff
@@ -11,40 +11,171 @@
...
+          <p class="aboutdetail">
+            Hello I’m a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+          <p class="connect">LET’S CONNECT</p>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use a header tag hx for the title.

File: index.html [hunk 6]
```diff
@@ -11,40 +11,171 @@
...
+            <li>
+              <a href="#"><i class="fa-brands fa-github fa-lg"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-solid fa-hand-peace fa-lg"></i></a>
+            </li>
+          </ul>
+          <button class="aboutresumebutton">Get my resume</button>
+          <ul class="techstack">
+            <li><p class="aboutlanguage">Languages</p><i class=" langicon fa-solid fa-chevron-down"></i></li>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly put all the frameworks and technology in a ul, each of them in a li

File: style.css [hunk 7]
```diff
@@ -125,3 +150,281 @@ ul.buttongroup > li > a:active {
...
+  margin-top: 16px;
+  margin-left: 16px;
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+
+/* Style card Properties */
+
+.workscard {
+  display: flex;
```
Comment (Kossi-stack) [inline style.css]: - [x] Kindly make sure the main container of the projects has a display:grid

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @zewdu444,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (zewdu444) [reply index.html]: @Kossi-stack thank you for the feedback  changed to section tag element.

File: index.html (hunk 2 above)
Comment (zewdu444) [reply index.html]: @Kossi-stack thank you for the feedback  changed to h2 tag element.

File: index.html (hunk 3 above)
Comment (zewdu444) [reply index.html]: @Kossi-stack thank you for the feedback  changed to section tag element

File: index.html (hunk 4 above)
Comment (zewdu444) [reply index.html]: @Kossi-stack thank you for the feedback  changed to h2 tag element.

File: index.html (hunk 5 above)
Comment (zewdu444) [reply index.html]: @Kossi-stack thank you for the feedback  changed to h2 tag element.

File: style.css (hunk 7 above)
Comment (zewdu444) [reply style.css]: @Kossi-stack thank you for the feedback  project showcase container were updated to gird

File: index.html (hunk 6 above)
Comment (zewdu444) [reply index.html]: @Kossi-stack I appreciate your evaluation. As you noted, the language list was divided into sub-lists, and the main list contained the language, framework, and technology.

**Pull request merged**